techniques/android/MASTG-TECH-0142.md

```diff
@@ -0,0 +1,195 @@
+
+### Data Types
+
+Apps may collect the following types of data:
+
+- **Location**: Approximate location, Precise location
+- **Personal info**: Name, Email address, User IDs, Address, Phone number, Race and ethnicity, Political or religious beliefs, Sexual orientation, Other personal info
+- **Financial info**: User payment info, Purchase history, Credit score, Other financial info
+- **Health and fitness**: Health info, Fitness info
+- **Messages**: Emails, SMS or MMS, Other in-app messages
+- **Photos and videos**: Photos, Videos
+- **Audio files**: Voice or sound recordings, Music files, Other audio files
+- **Files and docs**: Files and docs
+- **Calendar**: Calendar events
+- **Contacts**: Contacts
+- **App activity**: App interactions, In-app search history, Installed apps, Other user-generated content, Other actions
+- **Web browsing**: Web browsing history
+- **App info and performance**: Crash logs, Diagnostics, Other app performance data
+- **Device or other IDs**: Device or other IDs
+
+### Security Practices
+
+The Data Safety section indicates whether the app:
+
+- Encrypts data in transit
+- Encrypts data at rest
+- Provides a way for users to request data deletion
+- Follows the [Play Families Policy](https://support.google.com/googleplay/android-developer/answer/9893335)
+- Has committed to following Google Play's [data safety requirements](https://support.google.com/googleplay/android-developer/answer/11971187)
+- Has had an independent security review
+
+### Data Usage and Handling
+
+For each data type collected, the app should specify:
+
+- **Collection**: Whether the data is collected
+- **Sharing**: Whether the data is shared with third parties
+- **Purpose**: Why the data is collected (app functionality, analytics, advertising, fraud prevention, personalization, account management)
+- **Optional**: Whether data collection is optional or required for app functionality
+
+## Validation
+
+You should use the Data Safety section as a starting point for testing, not as definitive truth. During your security assessment:
+
+1. Verify that all data types claimed to be collected are actually being collected
+2. Identify any data types being collected that are not declared
+3. Confirm that security practices (encryption, deletion capabilities) are properly implemented
+4. Test whether "optional" data collection can truly be avoided
+5. Verify that data sharing claims match actual network traffic analysis
+
+Common issues to look for:
+
+- Apps collecting sensitive data not declared in the Data Safety section
+- Claims of encryption that are not implemented or are improperly configured
+- Missing functionality for data deletion despite claims in the Data Safety section
+- Data sharing with third parties not disclosed
+- Required data collection marked as optional
+
+## Caveats and Limitations
```

Comment:
It is not possible to distinguish between what is being declared by SDKs and the main app. However, the developer of the main app is held responsible as soon as they integrate an SDK.